test/plugin/scenarios/shardingsphere-3.x-scenario/src/main/java/org/apache/skywalking/apm/testcase/shardingsphere/service/api/service/CommonServiceImpl.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.testcase.shardingsphere.service.api.service;
+
+import org.apache.skywalking.apm.testcase.shardingsphere.service.api.entity.Order;
+import org.apache.skywalking.apm.testcase.shardingsphere.service.api.entity.OrderItem;
+import org.apache.skywalking.apm.testcase.shardingsphere.service.api.repository.OrderItemRepository;
+import org.apache.skywalking.apm.testcase.shardingsphere.service.api.repository.OrderRepository;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class CommonServiceImpl implements CommonService {
+    
+    @Override
+    public void initEnvironment() {
+        getOrderRepository().createTableIfNotExists();
+        getOrderItemRepository().createTableIfNotExists();
+        getOrderRepository().truncateTable();
+        getOrderItemRepository().truncateTable();
+        insertData();
+    }
+    
+    @Override
+    public void cleanEnvironment() {
+        getOrderRepository().dropTable();
+        getOrderItemRepository().dropTable();
+    }
+    
+    @Override
+    public void processSuccess(final boolean isRangeSharding) {
+        System.out.println("-------------- Process Success Begin ---------------");
+        printData(isRangeSharding);
+        System.out.println("-------------- Process Success Finish --------------");
```

Comment:
We don't need those. Test-framework does not get `SystemOut` log, so those are unuseful.

Reply:
Git it.